_freeze/location-services/read-data/execute-results/html.json

```diff
@@ -1,8 +1,8 @@
 {
-  "hash": "bee19783ff33107fa807b330c17c1550",
+  "hash": "2489724dc99bbf1e47c05af4dbdc6f70",
```

Comment:
@mmachir the way I've designed the site for now is that we can render the pages locally and they are instructed to "Freeze". So the rendered markdown is stored into a json file that is used to generate the website. This way the automatic job by GitHub actions can rebuild and publish the site without having to install R or the packages. Can probably fix that later. 

But the point is that this file is just the rendered output of the `read-data.qmd` file.

Reply:
Thanks for explaining this!!